### PR TITLE
feat: added gaming reference app to generate-application

### PR DESCRIPTION
**Pull request by ezgi-affinidi**

### CODE REVIEW

File: src/commands/generate-application/index.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@ export enum Platforms {
 }
 
 export enum UseCasesAppNames {
-  portableReputation = 'portable-reputation',
+  gaming = 'gaming',
+  career = 'career',
```
Comment (vitalyrudenko-affinidi) [inline src/commands/generate-application/index.ts:19]: Other use cases have "ReferenceApp" suffix. Let's be consistent with them (even tho it's a bit ugly).

File: src/render/texts.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import chalk from 'chalk'
 
+const isReputationApp = (useCase: string) => ['career', 'gaming'].includes(useCase)
```
Comment (vitalyrudenko-affinidi) [inline src/render/texts.ts:3]:
```suggestion
const isPortableReputationReferenceApp = (useCase: string) => ['career', 'gaming'].includes(useCase)
```

File: src/commands/generate-application/index.ts (hunk 1 above)
Comment (vitalyrudenko-affinidi) [reply src/commands/generate-application/index.ts:19]: In other places it's changed, but this file is not changed 🤔

**Review by dendavidov-affinidi — APPROVED** [review]

**Pull request merged**